### DISCUSSION TITLE
Address editorialNote on Contracts/hasGoverningJurisdiction

### DISCUSSION THREAD

**Discussion by rivettp** [edmcouncil/fibo#1385] [General]
There may be the need for 2 properties - one for where the contract is in force, the other for dispute resolution. Even if not, the note needs to be removed. 
I think we need a new issue - it's not clear whether we need a second property.  My understanding is that the jurisdiction is the one whose laws govern how any dispute will be handled and when required, where the contract is legally recorded

_Originally posted by @ElisaKendall in https://github.com/edmcouncil/fibo/diffs/15_

**Comment by trypuz** [comment]
See "[has governing jurisdiction](https://spec.edmcouncil.org/fibo/ontology/FND/Agreements/Contracts/hasGoverningJurisdiction)" property in FIBO Viewer.
